### PR TITLE
update build zon path

### PR DESCRIPTION
**Pull request by 0xycvv**
Update the build api to compatible with zig 0.13.0.

### CODE REVIEW

File: build.zig.zon [hunk 1]
```diff
@@ -1,6 +1,10 @@
 .{
     .name = "zig-base32",
     .version = "0.1.0",
```
Comment (0xycvv) [inline build.zig.zon]: lmk if we need to bump this

Comment (jsign) [reply build.zig.zon]: Sure, let's bump it to `v0.2.0` maybe to signal a breaking change.

Comment (jsign) [reply build.zig.zon]: Do you mind also bumping the zig version [in the CI](https://github.com/jsign/zig-base32/blob/main/.github/workflows/ci.yml) so we can have a green check? 

If you get in a rabbit hole don't worry and I can merge the PR and fix after.

**Review by jsign — APPROVED** [review]
Thank you!

**Pull request merged**